translations/ES_eu (international Spanish really)

```diff
@@ -0,0 +1,128 @@
+      nodes: "Nodos"
+      wallets: "Monederos"
+      community: "Comunidad"
+      services: "Servicios"
+      projects: "Proyectos"
+      exchanges: "Casas de cambio"
+      faq: "FAQ"
+    layout:
+      bitcoin_cash: "Bitcoin Cash"
+      lead: "Dinero electrónico de usuario a usuario"
+      loading: "Cargando..."
+    features:
+      - title: "rápido"
+        description: "Transacciones en segundos, confirmaciones en minutos."
+      - title: "Fiable"
+        description: "Una red sin congestión"
+      - title: "Low Fees"
+        description: "Envía dinero a cualquier parte por céntimos."
+      - title: "Simple"
+        description: "Fácil de usar y sin complicaciones."
+      - title: "Estable"
+        description: "Un sistema de pago que demuestra ser una reserva de valor."
+      - title: "Seguro"
+        description: "La tecnología blockchain más robusta del mundo."
+    headings:
+      about: "El mejor dinero del mundo"
+      features: "Nuevas características"
+      nodes: "Nodos"
+      wallets: "Monederos"
+      services: "Servicios"
+      projects: "Proyectos"
+      exchanges: "Casas de cambio"
+      faq: "Preguntas frecuentes"
+    content:
+      about:
+        - |
+          Bitcoin Cash ofrece una moneda sólida al mundo, cumpliendo con la promesa inicial de Bitcoin como "Dinero electrónico Peer-to-Peer" (de usuario a usuario). Usuarios y comerciantes pueden disfrutar de comisiones bajas y confirmaciones fiables. El futuro es prometedor gracias a crecimiento sin restricciones, adopción global, innovación que no necesita permisos y desarrollo descentralizado.
+        - |
+          Todo aquel que poseía Bitcoin antes del bloque 478558 también es poseedor de Bitcoin Cash. Todo el mundo es bienvenido en la comunidad de Bitcoin Cash a medida que avanzamos en la creación de dinero sólido al alcance de todas las personas del planeta.
+      features:
+        - |
+          <strong>Escalabilidad on-chain o en la cadena</strong> - Bitcoin Cash se ciñe a la hoja de ruta de Nakamoto de conseguir la adopción global al escalar <i>on-chain</i> (en la cadena). Como primer paso se ha implementado un límite del tamaño del bloque ajustable, con una capacidad estándar aumentada a 8MB. <a href="https://github.com/BitcoinUnlimited/BUIP/blob/master/065.mediawiki" target="_blank">Se está estudiando el desarollo que permita aumentos masivos en el futuro</a>.
+        - |
+          <strong>Nuevas firmas digitales</strong> - Un nuevo tipo de SigHash proporciona <i>replay protection</i> (protección frente a transacciones repetidas), mayor seguridad en monederos hardware, y la eliminación del problema de hashing cuadrático.
+        - |
+          <strong>Nuevo algoritmo de ajuste de la dificultad (DAA)</strong> - Un ajuste de la dificultad de <i>Proof-of-Work</i> reactiva permite a los mineros migrar desde la cadena antigua de Bitcoin (BTC) según lo deseen, a la vez que ofrece protección frente a fluctuaciones en la tasa de hash.
```

Comment:
I opted to leave DAA because it's a known term that shows up in many forums, so readers can identify it. It would make no sense to "create" the siglas AAD out of the blue, visible for the first time in this page. Kind of like GM (general motors) ---> no "MG" in a spanish translation.

Reactivo, you are right, that was a mistake on my part.

Lastly, what do you think about "fluctuaciones en el _hashrate_ (potencia de minado)."

Reply:
I don't feel strongly about the DAA thing but slightly favor AAD. GM seems kind of like a special case since it's a brand and the logo is precisely GM. An example for the opposite would be IMF spelled as FMI in Spanish (USA -> EUA, and other examples).

Reply:
I advise strongly against using AAD. During my career as a professional translator, I am never allowed to create new "siglas". Do you agree about removing the parenthesis and the "siglas" entirely?

Reply:
OK, we better trust your judgement then :)